### PR TITLE
Remove the BrowserLink submodule

### PR DESCRIPTION
**Pull request by natemcmaster**
This removes the submodule to https://github.com/aspnet/BrowserLink. This repo will build on its own, separately from this repo.

Resolves https://github.com/aspnet/AspNetCore-Internal/issues/1340

### CODE REVIEW

**Review by ryanbrandenburg — APPROVED** [review]

**Comment by natemcmaster** [comment]
Will wait for @Eilon's approval on this too before removing BrowserLink from our build.

**Review by Eilon — APPROVED** [review]

**Pull request merged**
